packages/loader/container-loader/src/container.ts

```diff
@@ -1842,4 +1862,65 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     private logContainerError(warning: ContainerWarning) {
         this.logger.sendErrorEvent({ eventName: "ContainerWarning" }, warning);
     }
+
+    private async loadContainerCopy(
+        // back-compat 0.35 stop passing loader post 0.35 support
+        loader: Loader,
+        urlResolver: IUrlResolver,
+        baseRequestUrl: string | undefined,
+        clientDetails?: IClientDetails,
+        fromSequenceNumber?: number,
+        summarizingClient?: boolean,
+        // TODO #4912 eventually we may wish to spawn an execution context from which to run this
+        executionContext?: string,
```

Comment:
Unused param, should this be removed?  Or are you planning to hook this up in a subsequent change?

Reply:
Yep planning to hook it up as part of the issue to do background summarization.  I guess since it's an optional anyway it's safer just to omit it for now in case we decide to change how it looks later